safe_transaction_service/tokens/services/price_service.py

```diff
@@ -30,8 +31,8 @@
     PricePoolOracle,
     SushiswapOracle,
     UnderlyingToken,
-    UniswapOracle,
```

Comment:
`UniswapOracle` was removed. Was this intended?

Reply:
Yes, it's a very old oracle, there's already Uniswap V2 and Uniswap V3